advanced_functionality/multi_model_catboost/multi_model_catboost.ipynb

```diff
@@ -0,0 +1,550 @@
+{
```

Comment:
<p>Typo: <code>moels</code> -&gt; <code>models</code>   </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3508/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='35'/>

Reply:
<p>Fixed in the latest push</p>

Reply:
fixed